stdlib/builtin/builtin.rbs

```diff
@@ -35,9 +35,6 @@ interface _Exception
                | (String arg0) -> Exception
 end
 
-class BigDecimal
```

Comment:
How about keeping the `BigDecimal` definition here? It will make `math.rbs` valid even without `bigdecimal` library. 

```
class BigDecimal < Numeric
  # You can find the definition of BigDecimal in bigdecimal/bigdecimal.rbs.
  # This is a dummy definition for Math module.
end
```

(Another way is to edit `math.rbs` to remove all `BigDecimal` references, and add `Math` methods overloadings in `bigdecimal` directory. I don't think this makes sense.)

Reply:
I replaced `Integer | Float | Rational | BigDecimal` in `math.rbs` to `Numeric` on unpushed commit. [These args are applied `numeric_to_float` function](https://github.com/ruby/ruby/blob/0b81a48/object.c#L3829) in [their implementations](https://github.com/ruby/ruby/blob/0b81a48/math.c#L78). So I think `Numeric` is suitable for these types. How is this?

Reply:
Good! It makes more sense! 👏